lm_eval/tasks/__init__.py

```diff
@@ -111,6 +112,22 @@
     # XQuAD
     "xquad_en": xquad.XQuADEnglish,
     "xquad_ar": xquad.XQuADArabic,
+    # XNLI
+    "xnli_en": xnli.XNLIEn,
+    "xnli_fr": xnli.XNLIFr,
+    "xnli_es": xnli.XNLIEs,
+    "xnli_de": xnli.XNLIDe,
+    "xnli_el": xnli.XNLIEl,
+    "xnli_bg": xnli.XNLIBg,
+    "xnli_ru": xnli.XNLIRu,
+    "xnli_tr": xnli.XNLITr,
+    "xnli_ar": xnli.XNLIAr,
+    "xnli_vi": xnli.XNLIVi,
+    "xnli_th": xnli.XNLITh,
+    "xnli_zh": xnli.XNLIZh,
+    "xnli_hi": xnli.XNLIHi,
+    "xnli_sw": xnli.XNLISw,
+    "xnli_ur": xnli.XNLIUr,
```

Comment:
Remove these tasks (see comment above about lack of `promptsource` support for non-English tasks).